react-toggle-switch-component/my-app/src/ToggleButton.tsx

```diff
@@ -0,0 +1,26 @@
+import { useState } from 'react';
+import './ToggleButton.css';
+
+export function ToggleButton() {
+  const [toggle, setToggle] = useState(true);
+
+  function handleToggle() {
+    setToggle(!toggle);
+    const slider = document.querySelector('#slider');
```

Comment:
No! You should not ever use `document.querySelector` in a React program!

You should set the className in the JSX using JavaScript variables.